packages/nimble-components/src/select/tests/select.stories.ts

```diff
@@ -96,6 +101,14 @@ const metadata: Meta<SelectArgs> = {
             filter-mode="${x => (x.filterMode === 'none' ? undefined : x.filterMode)}"
             style="width: var(${menuMinWidth.cssCustomProperty});"
         >
+            ${when(x => x.placeholder, html`
+                <${listOptionTag}
+                    disabled
+                    selected
+                    hidden>
+                    Select an option:
```

Comment:
Assuming we agree on the guidance in the other thread, we should remove the colon
```suggestion
                    Select an option
```